src/main/scala/com/stratio/hermes/helpers/ResourcesHelper.scala

```diff
@@ -0,0 +1,37 @@
+
+package com.stratio.hermes.helpers
+
+import java.io.{File, InputStream}
+
+import com.stratio.hermes.exceptions.HermesException
+import com.stratio.hermes.implicits.HermesSerializer
+import org.json4s.native.Serialization.read
+
+import scala.util.{Failure, Success, Try}
+
+object ResourcesHelper extends HermesSerializer {
+
+  def getResources(name: String): Seq[String] = Try(
+    new File(getClass.getResource(s"/locales/$name").getFile)) match {
+    case Success(resources) => resources.list().toSeq
+    case Failure(_) => throw new HermesException(s"Error loading invalid name /locales/$name")
```

Comment:
If we don't use the exception within the Failure it might be better to use the getOrElse way. Have a look at the following url and let me know what you think: [Error Handling With Try](http://danielwestheide.com/blog/2012/12/26/the-neophytes-guide-to-scala-part-6-error-handling-with-try.html)